### PR TITLE
apparently branching from main was the wrong option

### PR DESCRIPTION
**Pull request by sgosline**
Going to prune dead branches to avoid this in the future. dont merge yet

### CODE REVIEW

File: build/beatAML/GetBeatAML.py
Comment (jjacobson95) [inline build/beatAML/GetBeatAML.py:1]: A couple notes on things that may break the schema.
- Looks like model type was changed from "tumor" to "ex vivo".
- The variant classification mapping dictionary was removed.

I can update the CI to use this to generate the drugs file. Which is cleaner than using [beataml_drug_retrieval.py](https://github.com/PNNL-CompBio/coderdata/pull/110/files#diff-ec0ed3c05a58477f09cf2298128c42f2dd63bbd6862504ab55fe74a2a6de9302).

**Comment by sgosline** [comment]
Do NOT modify the CI at this point - everything should be running in the local script BEFORE it goes into CI to ensure reproducibility of the individual modules. the drugs files are generated using python scripts/build_locally.py --drugs

File: build/beatAML/GetBeatAML.py
Comment (jjacobson95) [reply build/beatAML/GetBeatAML.py:1]: This version doesn't call pubchem_retrieval, so we do have some duplicate code between files now as well. I'll add a note to update that as well.

**Comment by sgosline** [comment]
please file issues for the above comments, commenting on a close PR will not fix them.

**Pull request merged**